README.md

```diff
@@ -55,39 +55,80 @@ page.to_h
 # => { :title => "Example Domain", :more_information => "http://www.iana.org/domains/reserved" }
 ```
 
-## Archiving scraped pages
+## Extending
 
-The default strategy for retrieving pages is `Scraped::Strategy::LiveRequest`. If you'd also like to archive a copy of the page you're scraping into a git branch, you can pass a `:strategy` option to the constructor:
+There are two main ways to extend `scraped` with your own custom logic - custom requests and decorated responses. Custom requests allow you to change where the scraper is getting its responses from, e.g. you might want to make requests to archive.org if the site you're scraping has disappeared. Decorated responses allow you to manipulate the response before it's passed to the scraper, for example you might want to make all the links on the page absolute rather than relative.
+
+### Custom request strategies
+
+To make a custom request you'll need to create a class that subclasses `Scraped::Request::Strategy` and defines a `response` method.
 
 ```ruby
-ExamplePage.new(url: 'http://example.com', strategy: Scraped::Strategy::LiveRequestArchive.new)
+class FileOnDiskRequest < Scraped::Request::Strategy
+  def response
+    Response.new(url: url, body: open(filename).read)
+  end
+
+  private
+
+  def filename
+    @filename ||= File.join(URI.parse(url).host, Digest::SHA1.hexdigest(url))
+  end
+end
 ```
 
-This will use the [`scraped_page_archive`](https://github.com/everypolitician/scraped_page_archive)
-gem to store a copy of the pages you scrape in a git branch in your scraper's repo.
+The `response` method should return a `Response` instance. You need to pass at least `url` and `body` parameters to the `Response` constructor, you can also pass `status` and `headers` parameters.
+
+To use a custom request strategy pass it to `Scraped::Request`:
 
-## Custom strategies
+```ruby
+request = Scraped::Request.new(url: 'http://example.com', strategies: [FileOnDiskRequest, Scraped::Request::Strategy::LiveRequest])
+page = MyPersonPage.new(response: request.response)
+```
 
-If for some reason you can't scrape a site using one of the built in strategies
-then you can provide your own strategy.
+### Decorated responses
 
-A strategy is an object that implements a `response` method which returns a
-`Hash` with at least a `:body` key and optionally `:status` and `:headers` keys.
+To manipulate the response before it is passed to the scraper create a class that subclasses `Scraped::Response::Decorator`:
```

Comment:
I think this needs a second half that explains up-front what you need to do/provide/return with your subclass, e.g. "… and provides a replacement `body`, `url`, `status`, and/or `headers`")